src/imgix-core-js.js

```diff
@@ -214,6 +218,12 @@
       }
     };
 
+    function validateWidthTolerance(widthTolerance) {
+      if (typeof widthTolerance != 'number' || widthTolerance < 0) {
```

Comment:
We don't want someone passing a value greater than 1, do we? I don't think it would break anything, but would we ever want to encourage that?

```suggestion
      if (typeof widthTolerance != 'number' || widthTolerance < 0 || widthTolerance > 1) {
```

Reply:
No I don't think we would ever encourage that. But I am also hesitant to strictly forbid it. 1) I don't want to make assumptions about anyone's use case so as to say they could never want it and 2) As you said nothing will break, but it will probably not be an optimal solution. I think that's where we have drawn the line with validations in the past.